components/brave_rewards/browser/logging_util.cc

```diff
@@ -0,0 +1,136 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_rewards/browser/logging_util.h"
+
+#include "brave/components/brave_rewards/browser/file_util.h"
+
+#include "base/i18n/time_formatting.h"
+#include "base/files/file_path.h"
+#include "base/logging.h"
+#include "base/strings/stringprintf.h"
+#include "base/strings/utf_string_conversions.h"
+
+namespace brave_rewards {
+
+namespace {
+
+bool CreateLog(
+    base::File* file,
+    const base::FilePath& path) {
+  DCHECK(file);
+
+  file->Initialize(path, base::File::FLAG_CREATE_ALWAYS |
+      base::File::FLAG_READ | base::File::FLAG_WRITE | base::File::FLAG_ASYNC);
+
+  return file->IsValid();
+}
+
+bool OpenLog(
```

Comment:
where are we closing the file?

Reply:
The file is closed when the `base::File` object is destroyed